scripts/globals/magic.lua

```diff
@@ -569,9 +569,21 @@ function getSpellBonusAcc(caster, target, spell, params)
     --add acc for BLM AMII spells
     magicAccBonus = magicAccBonus + params.AMIIaccBonus;
 
-    --add acc for skillchains
-    if (skillchainTier > 0) then
-        magicAccBonus = magicAccBonus + 25;
+    -- Add acc for skillchains
+    local casterLevel = caster:getMainLvl()
+
+    if skillchainTier > 0 then
```

Comment:
I'm a little confused. Where is this info being pulled from?

Reply:
The figures? Nowhere, literally out of my behind. Just a general feeling I've heard from lots of DSP server players that Magic Burst accuracy bonus feels very underwhelming on DSP, and a static+25 all the way to 75 sort of supports that sentiment. The tiers and new figures I just made up based on what a few people suggested I try yesterday on Discord (instead of just changing the 25 to a bigger value and calling it good).

Reply:
I don't think retail even does a macc bonus like we have here at 25. I think its the mobs resistance to the SC's elements that is changed on retail, during the SC. Which isn't as similar as one might think.